recipes/libdispatch/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ex
+
+cmake ${CMAKE_ARGS} \
+    -G Ninja \
+    -B build \
+    -D BUILD_SHARED_LIBS=YES \
+    -D CMAKE_BUILD_TYPE=Release \
+    -D CMAKE_C_COMPILER=clang \
+    -D CMAKE_CXX_COMPILER=clang++ \
```

Comment:
This should be compilers that contain a host triple. Please add the following to conda_build_config.yaml
```suggestion
c_compiler:  # [linux]
 - clang  # [linux]
cxx_compiler:  # [linux]
 - clang  # [linux]
```

Reply:
Changes have been applied as per the code review, however, it seems the compiler supported in the CI is forced to gcc and becuase of this failure happens on linux CI after clang is specified.